File: Service_eHospital_DongNai_A/Implement/DM_MauDienCo_MauBenhService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class DM_MauDienCo_MauBenhService : BaseService<DM_MauDienCo_MauBenh, IDM_MauDienCo_MauBenhRepository>
    , IDM_MauDienCo_MauBenhService
    {
    private readonly IDM_MauDienCo_MauBenhRepository _DM_MauDienCo_MauBenhRepository;
    public DM_MauDienCo_MauBenhService(IDM_MauDienCo_MauBenhRepository DM_MauDienCo_MauBenhRepository) : base(DM_MauDienCo_MauBenhRepository)
    {
    _DM_MauDienCo_MauBenhRepository = DM_MauDienCo_MauBenhRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/DM_MauDienCo_MauBenh.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DM_MauDienCo_MauBenh } from './DM_MauDienCo_MauBenh.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DM_MauDienCo_MauBenhService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MauDienCo_MauBenh_Id', 'MauDienCo_Id', 'MauBenh_Id'];

    List: DM_MauDienCo_MauBenh[] | undefined;
    ListFilter: DM_MauDienCo_MauBenh[] | undefined;
    FormData!: DM_MauDienCo_MauBenh;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DM_MauDienCo_MauBenh";
    }
}

